### PR TITLE
Core/Packet: Chat

### PR DESCRIPTION
**Pull request by IntelligentQuantum**
## Changes Proposed:
-  Class Base
-

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Winfidonarleyan — APPROVED** [review]

File: src/server/game/Entities/GameObject/GameObject.cpp [hunk 1]
```diff
@@ -2100,7 +2100,7 @@ bool GameObject::IsInRange(float x, float y, float z, float radius) const
 }
 
 // pussywizard!
```
Comment (claudiodfc) [inline src/server/game/Entities/GameObject/GameObject.cpp:2102]: can these comments be removed?

File: src/server/game/Entities/Object/Object.cpp [hunk 2]
```diff
@@ -2196,7 +2196,7 @@ void Unit::BuildHeartBeatMsg(WorldPacket* data) const
 }
 
 // pussywizard!
```
Comment (claudiodfc) [inline src/server/game/Entities/Object/Object.cpp:2198]: can these comments be removed?

File: src/server/game/Entities/Player/Player.cpp [hunk 3]
```diff
@@ -5498,7 +5498,7 @@ void Player::SaveRecallPosition()
 }
 
 // pussywizard!
```
Comment (claudiodfc) [inline src/server/game/Entities/Player/Player.cpp:5500]: same here, whatever :)

File: src/server/game/Entities/Player/Player.cpp [hunk 4]
```diff
@@ -5511,7 +5511,7 @@ void Player::SendMessageToSetInRange(WorldPacket* data, float dist, bool self, b
...
```
Comment (claudiodfc) [inline src/server/game/Entities/Player/Player.cpp:5513]: again?

**Comment by claudiodfc** [comment]
Can you fix the merge conflicts? @IntelligentQuantum

**Comment by IntelligentQuantum** [comment]
@claudiodfc Done

**Comment by claudiodfc** [comment]
Test results ✔
- Used `say` chat. No issues.
- Used `yell` chat. No issues.
- Used `zone` chat. No issues.
- Used `guild` chat. No issues.
- Used `officer` chat. No issues.
- Used `whisper` chat. No issues.

**Comment by Kitzunu** [comment]
Do you have a pr for the module fail?

**Comment by IntelligentQuantum** [comment]
> Do you have a pr for the module fail?

https://github.com/azerothcore/mod-cfbg/pull/89

**Comment by FrancescoBorzi** [comment]
> > Do you have a pr for the module fail?
> 
> [azerothcore/mod-cfbg#89](https://github.com/azerothcore/mod-cfbg/pull/89)

Merged and restarted the modules build here

**Pull request merged**